test/unit/lib/plugins/package/lib/packageService.test.js

```diff
@@ -546,41 +185,47 @@ describe('lib/plugins/package/lib/packageService.test.js', () => {
           },
         },
       });
+      serverless = serverlessInstance;
+
+      fnIndividualZippedFiles = await listZipFiles(
+        path.join(servicePath, '.serverless', 'fnIndividual.zip')
+      );
     });
 
     it('should exclude custom plugins localPath', () => {
-      // Confirm that in function (fnService) dedicated artifact that ".custom-plugins/index.js" is not packaged
+      expect(fnIndividualZippedFiles).to.not.include('.custom-plugins/index.js');
     });
 
-    it('should support `package.individually`', () => {
+    it('TODO: should support `package.individually`', () => {
       // Confirm there's a "functions.fnService.package.artifact" and "functions.fnIndividual.package.artifact"
+      expect(serverless.service.getFunction('fnIndividual').package.artifact).to.include(
```

Comment:
also looks great :+1: Let's also add another assertion for `fnService` function that it also has an individual package created ending with `fnService.zip` :+1:

Reply:
Indeed. Sorry, for some reason I wasn't able to get `package.artifact` from `serverless.service.getFunction('fnIndividual')` - it was a PEBKAC. I struggle with `fnService` though. 
I suspect adding definition with `fnService` to the `function` configuration block is required, e.g.:
```js
 fnService: {
   name: 'fnService',
 },
```

But then I end up with `artifact.zip` as `package.artifact`.

Reply:
I see, it looks like there's a slightly wrong setup defined for this test which makes it harder - let's remove `configExt.package.artifact` from `runServerless` setup for this `describe`. It will make all functions be built `individually` - when it was set, it was forcing all functions that didn't have `individually` specified on function level to use existing `artifact.zip`. Second thing, after removing that `package.artifact`, we need to add `awsRequestStubMap:mockedDescribeStacksResponse` to that `runServerless` run - please take a look at how it's added in other `runServerless` invocations - you should be able to just pass already defined `mockedDescribeStacksResponse`. After having that in place, you shouldn't have to explicitly add `fnService` definition to `configExt` as it's already defined in fixture and `expect(serverless.service.getFunction('fnService').package.artifact).to.include('fnService.zip');` should work just fine.

Let me know if you have any troubles with that and I'd be happy to help :raised_hands:

Reply:
@pgrzesik Sorry, I've totally missed that `configExt.package.artifact` is set 🤦.
It should be ready for the review now.

Reply:
Did you push changes? I'm not seeing the `expect(serverless.service.getFunction('fnService').package.artifact).to.include('fnService.zip');`

Reply:
@pgrzesik Whoops, sorry! Pushed just now.